v2/emailpassword/pre-built-ui/setup/frontend.mdx

```diff
@@ -13,6 +13,7 @@ import TabItem from '@theme/TabItem';
 import {Question, Answer}from "/src/components/question"
 import AppInfoForm from "/src/components/appInfoForm"
```

Comment:
these changes are good. However, there are a few other instances of yarn / npm being used with supertokens-auth-react - for examlpe in here: https://deploy-preview-590--admiring-bhabha-7b1be9.netlify.app/docs/emailpassword/nextjs/init.

Please make sure to cover all such cases.